### PR TITLE
Rewritten from scratch

### PR DESCRIPTION
**Pull request by Abbe98**
In addition to visual change:

 - Tries to check if the image is already imported.
 - Picks up location label from Mapzen.

It uses the same URL parameter and should be fully backwards capable.
image

### CODE REVIEW

File: simpleUpload/index.html [hunk 1]
```diff
@@ -1,96 +1,35 @@
...
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <meta name="description" content="Mapillary to Wikimedia Commons uploader">
-    <meta name="author" content="Lokal_Profil">
-    <link rel="icon" href="../../favicon.ico">
-    <title>Mapillary2Commons</title>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charset="utf-8" />
```
Comment (lokal-profil) [inline simpleUpload/index.html]: Strange mixture of indents

**Comment by Abbe98** [comment]
Tested in IE/Edge, Firefox, and Opera/Chrome.

File: simpleUpload/index.html [hunk 2]
```diff
@@ -1,96 +1,35 @@
...
-        </noscript>
-        <div class="container">
-            <div class="page-header">
-                <h1>Mapillary 2 Commons</h1>
-            </div>
-            <p class="lead">
-                Ever found a beautiful image on <a href="https://www.mapillary.com">Mapillary</a>
-                and thought <i>That would look great on <a href="https://commons.wikimedia.org">Wikimedia Commons</a></i>?
-            </p>
-            <p> Simply enter the image key or url below and click <i>Check!</i>.
```
Comment (lokal-profil) [inline simpleUpload/index.html:40]: Any reason why you completely dropped this information bit?

File: simpleUpload/index.html [hunk 3]
```diff
@@ -1,96 +1,35 @@
...
-                            You need a Wikimedia Commons account and to have
-                            <a href="https://tools.wmflabs.org/magnustools/oauth_uploader.php?action=authorize">authorized</a>
-                            OAuth Uploader for this tool to work.
-                        </li>
-                        <li>
-                            If Mapillary doesn't know the name of the location
-                            you'll be prompted to enter one.
-                        </li>
-                        <li>
-                            You can also pass this page a url directly using
```
Comment (lokal-profil) [inline simpleUpload/index.html:66]: Also this info

**Review by lokal-profil — COMMENTED** [review]
Sorry for taking so long to get back on this.

Thanks for giving this a work-over and making it work after the API deprecation.

A few inline questions. Also what happens if mapzen fails to find a location (i.e. are you stil prevented from going on without a location)?

**Comment by lokal-profil** [comment]
Temporarily up at https://tools.wmflabs.org/mapillary-commons/ver2/ for testing

File: simpleUpload/index.html (hunk 1 above)
Comment (Abbe98) [reply simpleUpload/index.html]: Noted

File: simpleUpload/index.html (hunk 2 above)
Comment (Abbe98) [reply simpleUpload/index.html:40]: No, not at all (in both cases), I started from scratch so I just didn't  think about it.

**Comment by Abbe98** [comment]
Adding the missing information and cleaning up the weird indents once I'm done for the day :-)

If Mapzen isn't able to (re)geocode the location, the user must enter a location description before the upload button gets enabled. Example: 
https://tools.wmflabs.org/mapillary-commons/ver2/?mapillary=https://www.mapillary.com/map/im/e814RJRxf8cnqn4OEFFMnQ

**Comment by lokal-profil** [comment]
@Abbe98

In my mobile at least if I click the Upload button it reloads the page.  Could it display a message asking for location input instead?

Also noticed that if I fill in a location and then erase it it still uses the first letters of the deleted string afterwards.

**Comment by Abbe98** [comment]
Manual location descriptions should work now and if the description is to short or does not exists a error message will replace the file name. (The Upload button does no longer reload the page.) 

The existing information about OAuth was moved to a fixed box, earlier you could miss it if the `mapillary` parameter was in use.

In the place of OAuth information there is now the old `lead` text.

I added a link to source code and API docs at the bottom of the header and changed the README to include information about the `mapillary` parameter.

**Comment by lokal-profil** [comment]
Thank you!

**Comment by lokal-profil** [comment]
Deployed

**Comment by Abbe98** [comment]
Amazing, thanks taking the time!

**Pull request merged**